README.md

```diff
@@ -56,6 +54,44 @@ brew install foundry
 
 Run `foundry version` to verify it's installed correctly.
 
+## Getting Started
+
+Clone this sample to your local system, or [download as a zip file](https://github.com/CrowdStrike/foundry-sample-scalable-rtr/archive/refs/heads/main.zip).
+
+```shell
+git clone https://github.com/mraible/foundry-sample-scalable-rtr
+cd foundry-sample-scalable-rtr
+```
+
+Log in to your Foundry instance:
```

Comment:
I think the `foundry login` command is named appropriately since it does log you in and configure the CLI to talk to a Foundry instance. Most of the CLIs I've worked with in the past use the same command. Both the Auth0 CLI and Okta CLI operate like this.

As a developer, what I found really useful is knowing the profiles are stored in `~/.config/foundry/configuration.yml` so I could manually edit them.

Reply:
well that's good cuz I'm sure we're not renaming the command! lol.

btw what do you mean by a `Foundry instance`? Wondering whether the docs already have a word for the same concept (but very possibly not).

That info about manually editing the config file should be doc'd - would you mind logging a PLATFORMPG docs task for that?

Reply:
Foundry instance = the cloud you're deploying to (US-1, US-2, or EU). Maybe this should be simplified to "Log in to Foundry:". I'll enter an issue for documenting how to edit the config file.

Reply:
ah OK, agreed on simplifying to "log in to Foundry"